net/net/src/TNetFile.cxx

```diff
@@ -694,7 +694,6 @@ void TNetFile::Create(const char * /*url*/, Option_t *option, Int_t netopt)
    Bool_t update   = (fOption == "UPDATE") ? kTRUE : kFALSE;
    Bool_t read     = (fOption == "READ") ? kTRUE : kFALSE;
    if (!create && !recreate && !update && !read) {
-      read    = kTRUE;
```

Comment:
This is one code pattern where I see "some" value in updating the variable even if it is, currently, not used.

If this routine is update at a later date, the developer might assume that the variable 'read' is true is all cases of 'reading mode' even the default one.

On the other hand, not having it at all might be event clearer.  i.e. except for the slight extra run-time cost, I think the fillowing code has the same net effect:
```
Bool_t update = (fOption == "UPDATE") ? kTRUE : kFALSE;
if (!create && !recreate && !update) {
     fOption = "READ";
}
```
with a spurrious re-assignment in case `fOption == "READ"` ... 
So actually maybe even:
```
Bool_t update = (fOption == "UPDATE") ? kTRUE : kFALSE;
if (!create && !recreate && !update && (fOption != "READ")) {
     fOption = "READ";
}
```

Reply:
I am not sure.
`fOption` can be accessed via `TFile::GetOption()` method - user can always check that was original option he/she provide. 
None of other TFile-based classes, including TFile itself, change fOption in such case.
Either we do it consistently in all places, or do not change it here.

Reply:
> None of other TFile-based classes, including TFile itself, change fOption in such case.

Sorry, I am wrong. I will update PR